Update french translation : yii.php

zoolanders49 opened the pull request:
add "must be equal to"  translation for french

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  |

zoolanders49 committed:
Update yii.php

add "must be equal to"  translation for french

cebe commented:
Thank you! There are other strings that are untranslated in French, could you provide translations for these too?

```
    ' and ' => '',
    'The combination {values} of {attributes} has already been taken.' => '',
```

zoolanders49 committed:
Update yii.php

add translation for :
' and ', 
'The combination {values} of {attributes} has already been taken.'

cebe commented:
Great, thanks!

zoolanders49 commented:
hello,  I add the two others translations.
